autoscheduler/frontend/src/components/NavBar.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import {
+  AppBar, Toolbar, Typography, Button, IconButton, makeStyles,
+} from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  menuButton: {
+    marginRight: theme.spacing(2),
+  },
+  title: {
+    flexGrow: 1,
+  },
+}));
+
+const NavBar: React.SFC = function App() {
+  const classes = useStyles(useStyles);
```

Comment:
How does this line work? Theoretically, you should be passing in a theme, but it seems like you're passing in a function instead

Reply:
I am defining the custom theme rules wanted to be used (line 6-16) and then passing it to be used (line 19).

Reply:
See #107 for how I think this should be done